docs/source/markdown/podman-run.1.md

```diff
@@ -924,7 +924,7 @@ container. Similarly, _source-volume_:_/container-dir_ will mount the volume
 in the host to the container. If no such named volume exists, Podman will
 create one.
 
-The _options_ is a comma delimited list and can be:
+The _options_ is a comma delimited list and can be: <sup>[[1]](#Footnote1)</sup>
```

Comment:
I don't think we need the footnote here.

Reply:
slave mounts show up in the lines right after this, but i dont think we should put the footnote there either since it would show up on the manpage funny. let me know if its better to just remove it from here or move it somewhere else

Reply:
No problem as long as you have a reason.